js/client.js

```diff
@@ -879,7 +879,9 @@ function toId() {
 						self.send('/join ' + roomid);
 					});
 				} else if (data === 'rename') {
-					this.renameRoom(roomid, errormessage);
+					// |newtitle|newid
```

Comment:
This should also be `|newid|newtitle`.

Unless it wasn't in the past?

Reply:
> Unless it wasn't in the past?

It didn't exist in the past. I just appended roomid to it in the server code when I realized it was necessary. Is there a difference?

Reply:
Was it accepting titles in the past?

Reply:
It seems like `|rename|newid` was the old signature. If you want to support titles, it should go after the id, not before.